src/ChipCarousel/README.md

```diff
@@ -0,0 +1,73 @@
+---
+title: 'ChipCarousel'
+type: 'component'
+components:
+- ChipCarousel
+categories:
+- Content
+status: 'New'
+designStatus: 'Done'
+devStatus: 'Done'
+notes: |
+---
+
+The ``ChipCarousel`` component creates a scrollable horizontal block of chips with buttons for navigating to the previous and next set of chips.
+
+## Basic Usage
+
+```jsx live
+() => {
+  const [offset, setOffset] = useState(120);
+  const [offsetType, setOffsetType] = useState('fixed');
+  const [gap, setGap] = useState(3)
+  
+  return (
+    <>
+      {/* start example form block */}
+      <ExamplePropsForm
+        inputs={[
+          {
+            value: offset,
+            setValue: setOffset,
+            range: {
+              min: 0,
+              max: offsetType === 'percentage' ? 100 : 1000,
+              step: offsetType === 'percentage' ? 1 : 50,
```

Comment:
On page load, `offsetType` is defaulted to "fixed" at a value of 120. However, if the user switches to "percentage", 120 > the `max` of 100, so the percentage offset changes to 100%, which doesn't quite demonstrate the percentage increment option effectively.

I wonder if we should reset the `offset` value when switching between `offsetType`s so the offset values make sense for each type upon switching?

Reply:
Added function to be more user-friendly when `offsetType` is switched. It calculates ration and sets relatively equal value between `fixed` and `percentage`.